packages/components/radio/src/radio.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface RadioProps {
+    shape?: 'circle' | 'square';
+    backgroundColor?: string;
+    label: string;
+    onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    checked?: boolean;
+    name: string;
+    value: string;
+    disabled?: boolean;
+  }
+
+  export interface RadioOption {
+    label: string;
+    value: string;
+    shape?: 'circle' | 'square';
+    backgroundColor?: string;
+    disabled?: boolean;
```

Comment:
If disabled is optional, it's a good practice to set the prop as false by default.